ynput/ayon-frontend
# Save context when refreshing page

## Changelog Description

- When the user refreshes the page the current state of the app should remain.
- Only works on editor, browser and workfiles.
- One negative is it makes the context slice extremely complicated, but it's very easy to now add states that sync with local storage.
- Fixes lots of issues with version select and improves how it fetches and caches version data.
- Adds a context menu on hierarchy that allows you to reset all products to latest version.

https://github.com/ynput/ayon-frontend/assets/49156310/14bc12ab-8de0-4e1a-8939-cd7040c1dee0

## Testing

1. Open some folders, select loads of stuff.
2. Refresh the page and see if everything's the same.
